Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/RustCrypto/RSA"
 keywords = ["rsa", "encryption", "security", "crypto"]
 categories = ["cryptography"]
 readme = "README.md"
-rust-version = "1.60"
+rust-version = "1.65"
```

Comment:
this is a little problematic, I have at least one downstream dependency that can't upgrade to 1.65 yet, due to compiler issues in the lates compiler versions :/ https://github.com/deltachat/deltachat-core-rust/pull/3856